package.json

```diff
@@ -94,20 +93,22 @@
   "dependencies": {
     "clsx": "^1.1.1",
     "lodash": "^4.17.20",
+    "nebenan-eventproxy": "^1.0.0",
```

Comment:
`eventproxy` and `keymanager` need to be peer dependencies. Running multiple versions of `eventproxy` at the same time will defeat its purpose (it's already a peerDep, so you're good)